lib/client.js

```diff
@@ -7,8 +7,21 @@ var OKCupid = function(){
 }
 
 OKCupid.prototype.login = function(username, password, callback){
-    var login_form = {username: username, password: password};
-    this.requester.postRequest(URLS.login, login_form, callback)
+    var login_form = {
+      okc_api: 1, 
+      username: username, 
+      password: password,
+    };
+    this.requester.postRequest(URLS.login, login_form, function(err, res, body) {
+      // if there is a non-zero status in response and response does not show an error
+      if (body.status !== 0 && !err) {
```

Comment:
Should you check for `!err` first before checking `body.status`?
`!err && body.status !== 0`

Allows the logic to short circuit on error, and that you don't get unknown type error if `body` is undefined`.